tests/integ/modin/groupby/test_groupby_basic_agg.py

```diff
@@ -388,6 +389,18 @@ def test_string_sum(data):
     )
 
 
+@sql_count_checker(query_count=0)
+def test_groupby_sum_string_argument_exception():
```

Comment:
This test case seems redundant given that we have `test_groupby_agg_invalid_numeric_only`. Can we delete it?